state/remoteentities.go

```diff
@@ -22,6 +24,7 @@ type remoteEntityDoc struct {
 	SourceModelUUID string `bson:"source-model-uuid"`
 	EntityTag       string `bson:"entity"`
 	Token           string `bson:"token"`
+	Macaroon        string `bson:"macaroon,omitempty"`
```

Comment:
maybe store as []byte, and use the macaroon's BinaryMarshaler interface? should be more compact